media/doc/Romanian translation notes.txt

```diff
@@ -1,5 +1,8 @@
 The following material is addressed (in Romanian) to Romanian translators.
 
+Copyright 2017 ColinFinck
```

Comment:
It wasn't him who wrote the Romanian translation notes but Ștefan himself, see https://github.com/reactos/reactos/commit/c7a0e560751c91465529153e4f46ef17dc82163c

Reply:
I modified it. Sorry for the mistake. I didn't pay attention.